e2e/browser/test-app/components/accessGrants/index.tsx

```diff
@@ -80,7 +81,7 @@ export default function AccessGrant({
     setSharedResourceIri(getSourceUrl(savedFile));
   };
 
-  const handleDelete = async (e) => {
+  const handleDelete = async (e: any) => {
```

Comment:
Cam we use a more specific type than any?

Reply:
I think we could, but actually we don't need the event to go through at all if we just properly use HTML5 (as I learnt writing this code ^^): https://github.com/inrupt/solid-client-access-grants-js/pull/711/commits/d1d5a568a847da55c20ea0ed2e1299b390ad0c68